client/src/dummyImages.js

```diff
@@ -0,0 +1,25 @@
+const images = {
+  bannerImage: 'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/YzkyZGY0MDMt',
+  featuresBlock1logo: 'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/NmIwY2NkN2Ut',
+  featuresBlock1media: 'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/MzcwNDhhODct',
+  featuresBlock2: 'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/ZjkyM2M5OGQt',
+  featuresBlock3: 'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/MjQ5YWY4ZTYt',
+  featuresBlock4: 'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/OGFjNjQxZTMt',
+  featuresBlock5: 'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/ZWIxNGMwNmMt',
+  featuresBlock6: 'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/ODFhOWQzZDAt',
+  featuresBlock7: 'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/NDY0NjAwODYt',
+  featuresBlock8: [
+    'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/NGY2ODBhMGUt',
+    'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/Yzc5MTdiYTMt',
+    'https://m.media-amazon.com/images/G/01/kindle/journeys/lC800nCLcDMVzVWEilELmfZnKxn2B9aCBYWZKmCf8Okk3D/ZDU2MmU3YjEt'
+  ],
+  featuresBlock9: [
+    'https://m.media-amazon.com/images/G/01/kindle/mako/features/sustainability/materials._CB406459720_.png',
+    'https://m.media-amazon.com/images/G/01/kindle/mako/features/sustainability/energy._CB406459720_.png',
+    'https://m.media-amazon.com/images/G/01/kindle/mako/features/sustainability/lowpowermode._CB406459723_.png',
+    'https://m.media-amazon.com/images/G/01/kindle/mako/features/sustainability/alexa._CB406459720_.png',
+    'https://m.media-amazon.com/images/G/01/kindle/mako/features/sustainability/packing._CB406459720_.png'
+  ]
```

Comment:
Are these the images coming from Alek's API and you have configured test data for your module for unit testing?

Reply:
No, dummy data for now. Will be refactored when Alek's API is ready.